src/cdk-experimental/popover-edit/table-directives.ts

```diff
@@ -179,6 +193,35 @@ export class CdkPopoverEdit<C> implements AfterViewInit, OnDestroy {
         this.viewContainerRef,
         {$implicit: this.context}));
     this.focusTrap!.focusInitialElementWhenReady();
+
+    // Update the size of the popup initially and on subsequent changes to
+    // scroll position and viewport size.
+    merge(this.scrollDispatcher.scrolled(), this.viewportRuler.change())
+        .pipe(
+            startWith(null),
+            takeUntil(this.overlayRef!.detachments()),
+            )
+        .subscribe(() => {
+          this.overlayRef!.updateSize(
+              this.positionFactory.sizeConfigForCells(this._getOverlayCells()));
+        });
+  }
+
+  private _getOverlayCells(): HTMLElement[] {
+    const cell = closest(this.elementRef.nativeElement!, CELL_SELECTOR) as HTMLElement;
+
+    if (!this.colspan.before && !this.colspan.after) {
+      return [cell];
+    }
+
+    const row = closest(this.elementRef.nativeElement!, ROW_SELECTOR)!;
+    const rowCells = row.querySelectorAll(CELL_SELECTOR);
+    const ownIndex = Array.prototype.indexOf.call(rowCells, cell);
+
+    return Array.prototype.slice.call(
```

Comment:
Instead of using `.call` like this, you could do above
```ts
const rowCells = Array.from(row.querySelectorAll(CELL_SELECTOR));
```

Reply:
I'm aware of Array.from but it does not work in IE11 - is material/cdk using some kind of polyfill?

Reply:
Angular already relies on it, so we're safe to use it.